_includes/install/file-system-umask.md

```diff
@@ -7,22 +7,28 @@ We recommend changing the umask on a one-user or shared hosting system only. If
 
 The default umask (with no `magento_umask` specified) is `002`, which means:
 
-*	775 for directories, which means full control by the user, full control by the group, and enables everyone to traverse the directory. These permissions are typically required by shared hosting providers.
+* 775 for directories, which means full control by the user, full control by the group, and enables everyone to traverse the directory. These permissions are typically required by shared hosting providers.
 
-*	664 for files, which means writable by the user, writable by the group, and read-only for everyone else
+* 664 for files, which means writable by the user, writable by the group, and read-only for everyone else
 
 A common suggestion is to use a value of `022` in the `magento_umask` file, which means:
 
-*	755 for directories: full control for the user, and everyone else can traverse directories.
-*	644 for files: read-write permissions for the user, and read-only for everyone else.
+* 755 for directories: full control for the user, and everyone else can traverse directories.
+* 644 for files: read-write permissions for the user, and read-only for everyone else.
 
 To set `magento_umask`:
 
-1.	Log in to your Magento server, or switch to, the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
-2.	Set the value of `magento_umask`:
+1. In Terminal, log in to your Magento server, or switch to, the [Magento file system owner][].
```

Comment:
```suggestion
1. In a command line terminal, log in to your Magento server as a [Magento file system owner][].
```

Terminal is the name of a specific application. Suggest changing it to something more general.

Original sentence is confusing with the use of 'or'. Suggested change should make it more focused and clear.